recipes-kernel/linux/files/patches-6.1/0086-PCI-keystone-Fix-race-condition-when-initializing-PH.patch

```diff
@@ -0,0 +1,61 @@
+From 36836820f571872d6856c17bdc1d0e4f8418915d Mon Sep 17 00:00:00 2001
```

Comment:
The patch file should be generated from:

```shell
git format-patch --abbrev=12 --no-numbered --zero-commit --no-signature ...
```

see: https://github.com/siemens/meta-iot2050/blob/master/CONTRIBUTING.md#patch-files-generation

Meanwhile I will test the SSD on M.2 variant.

Reply:
Ok, I'll prepare the patches like that. Please let me know the results of your testing when you can so I can reply to TI

Reply:
Now with the patch my PM971 256G SSD is working.